src/LinuxTracing/LinuxTracingIntegrationTestPuppet.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef LINUX_TRACING_INTEGRATION_TEST_PUPPET_H_
+#define LINUX_TRACING_INTEGRATION_TEST_PUPPET_H_
+
+namespace orbit_linux_tracing {
```

Comment:
It would probably help to have a high level description here.

Reply:
There's a brief high level description in the .cpp file. Should I move it here? The reason why it's not here is that this `LinuxTracingIntegrationTestPuppet` is very similar to an executable (which would only have the .cpp), the header is mostly to share constants.

Reply:
Okay, I missed the one in the cpp file. Is fine for me.